packages/gatsby-remark-prismjs/src/index.js

```diff
@@ -79,7 +79,7 @@ module.exports = (
 
   if (!noInlineHighlight) {
     visit(markdownAST, `inlineCode`, node => {
-      let languageName = `text`
+      let languageName = `inline-text`
```

Comment:
It's probably a good idea to remove inline-text and replace it with text so we don't make a distinction. If people want to add their own custom language "text" they only have to do it once.

Reply:
I don't think it's worth it, they actually choose for this behaviour so they are probably smart enough to look at all code blocks. You might want to edit your warning based on the noInlineHighlight flag to include inline-code blocks in your message.

Reply:
@wardpeet I made those changes and a bit of extra test coverage to make sure the messages are tied to the noInlineHighlight flag.  I think this is a bit better.